components/atoms/Title/Title.tsx

```diff
@@ -6,6 +6,7 @@ import useTheme from '../../hooks/useTheme'
 export type TitleVariation =
   | 'title1'
   | 'title2'
+  | 'title4'
```

Comment:
Do we officially want to define a new title style? Did you confirm with Saaniya that our design convention will need this?

And if so, shouldn't we call it title3 rather than title4?

Reply:
safssf

Reply:
In the figma we have new style and its name is title4. However, let me confirm with Saaniya

Reply:
Just confirmed that she wanted to officially define a new style and yes it should be title 3. But the challenge is that a title 3 already exists and it’s being used through the app. To avoid confusion she labeled it as title 4 and she said if we could swap the 2 so it’s more logical, then that would be great.

Reply:
@jeansampen Where is a title3 being used throughout the app? Do you see reference to that elsewhere in lava-components? I don't think we have that. I'm not sure what she's referring to.

Reply:
@krschacht She says maybe it wasn’t updated in the component library or used. She has it in Figma for designs. She confirmed if that’s the case, we can go ahead and add the story description font as title 3 and she will go ahead and swap details in her Figma library